index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
+    </style>
+    <title>Prototype Mobile Portafolio</title>
+</head>
+<body>
+    <main class="container">
+        <header>
+            <h3>WELCOME</h3>
+            <section class="nav-bar">
+                <div class="line"><h3></h3></div>
+                <div class="line"><h3></h3></div>
+                <div class="line"><h3></h3></div>
+            </section>
+        </header>
+        <section class="presentation">
+            <h2 class="introduction">Hey there. I’m Rafael</h2>
```

Comment:
Kindly change this to be `h1` instead of `h2` for `Hey there. I’m Rafael` since this is the main heading of the page for semantic purposes. Remember that the `h1` should only exist once on your page.